getting-started/git-boot-camp.rst

```diff
@@ -283,6 +283,14 @@ Creating a Pull Request
 
 6. Press the ``Create pull request`` button.
 
+You should include the issue number in the title of the PR (``gh-NNNNN: ...``).
+
+You might also use `special keywords
+<https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword>`_
+to automatically close the linked issue once the PR is merged, however you
```

Comment:
```suggestion
to automatically close the linked issue once the PR is merged. However, you
```

Fix comma splice, add missing comma and this makes more sense as two separate sentences.